multi/imxrt-multi/imxrt-multi.c

```diff
@@ -207,9 +206,11 @@ static int createDevFiles(void)
 	oid_t dir;
 	char name[8];
 
-	while (lookup("/", NULL, &dir) < 0) {
-		usleep(100000);
-	}
+#if (BUILTIN_DUMMYFS) == 0
+	/* Wait for the filesystem */
+	while (lookup("/", NULL, &dir) < 0)
+		usleep(10000);
+#endif
```

Comment:
Uh, placement of this waiting loop is and was buggy, we create a file in `main()` before invoking `createDevFiles()`. Maybe move this to `main()`?

Reply:
Yes, exactly - I think so too. I'll move it to the right place.